workflow-scripts/update_lambda_template.py

```diff
@@ -0,0 +1,22 @@
+from ruamel.yaml import YAML
+
+yaml = YAML()
+
+file_path = 'lambda-template.yaml'
+
+with open(file_path, 'r') as file:
```

Comment:
instead of updating the same file, maybe create a prod folder and copy it there. Publish workflow can use templates from that folder.

Reply:
You mean copy of file local file for testing? Because if I change the path of production file it will cause issues now. And even if I create a different file for local testing then path in every other parent template files need to change which i think is a lot of changes. Currently with just update to lambda-template.yaml it will work as expected